assignments/game-of-life/code.js

```diff
@@ -0,0 +1,54 @@
+drawFilledRect(0, 0, width, height, 'black')
+
+
+
+const makeArrayRow = () => {
+  let row = []
+  for (let i = 0; i < width; i += 5) {
+    if (Math.random() < .2) {
+      row.push(1)
+    } else {
+      row.push(0)
+    }
+  }
+  return row
+}
+const makeArray = () => {
+  let b = []
+  for (let i = 0; i < height / 5; i++) {
+    b.push(makeArrayRow())
+  }
+  return b;
+}
+
+
+
+const drawTheThings = (b) => {
+  for (let i = 0; i < height / 5; i++) {
+    for (let j = 0; j < width / 5; j++) {
+      if (b[i][j] === 1) {
+        drawFilledRect(j * 5, i * 5, 5, 5, 'yellow')
```

Comment:
Now this line will only work if in fact the dimensions of `b` are 1/5th the dimensions of the drawing area. I might try to figure out the size of the cells here by dividing the `height` and `width` by the dimensions of `b` but more likely I'd just continue the pattern of punting decisions to our callers and add an argument to this function to specifie the `cellSize` and then change this line:

```suggestion
        drawFilledRect(j * cellSize, i * cellSize, cellSize, cellSize, 'yellow')
```